src/webhooks/fixtures/webhook.json

```diff
@@ -1 +1,67 @@
-{"id": "wh_123","data":{"id":"directory_user_01FAEAJCR3ZBZ30D8BD1924TVG","state":"active","emails":[{"type":"work","value":"blair@foo-corp.com","primary":true}],"idp_id":"00u1e8mutl6wlH3lL4x7","object":"directory_user","username":"blair@foo-corp.com","last_name":"Lunchford","first_name":"Blair","job_title":"Software Engineer","directory_id":"directory_01F9M7F68PZP8QXP8G7X5QRHS7","raw_attributes":{"name":{"givenName":"Blair","familyName":"Lunchford","middleName":"Elizabeth","honorificPrefix":"Ms."},"title":"Software Engineer","active":true,"emails":[{"type":"work","value":"blair@foo-corp.com","primary":true}],"groups":[],"locale":"en-US","schemas":["urn:ietf:params:scim:schemas:core:2.0:User","urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"],"userName":"blair@foo-corp.com","addresses":[{"region":"CA","primary":true,"locality":"San Francisco","postalCode":"94016"}],"externalId":"00u1e8mutl6wlH3lL4x7","displayName":"Blair Lunchford","urn:ietf:params:scim:schemas:extension:enterprise:2.0:User":{"manager":{"value":"2","displayName":"Kate Chapman"},"division":"Engineering","department":"Customer Success"}}},"event":"dsync.user.created","created_at":"2021-06-25T19:07:33.155Z"}
+{
+  "id": "wh_123",
+  "data": {
+    "id": "directory_user_01FAEAJCR3ZBZ30D8BD1924TVG",
+    "state": "active",
+    "emails": [
+      {
+        "type": "work",
+        "value": "blair@foo-corp.com",
+        "primary": true
+      }
+    ],
+    "idp_id": "00u1e8mutl6wlH3lL4x7",
+    "object": "directory_user",
+    "username": "blair@foo-corp.com",
+    "last_name": "Lunchford",
+    "first_name": "Blair",
+    "job_title": "Software Engineer",
+    "directory_id": "directory_01F9M7F68PZP8QXP8G7X5QRHS7",
+    "created_at": "2021-06-25T19:07:33.155Z",
```

Comment:
Just formatting to this file. The only change I actually made was adding `created_at` and `updated_at` which were added later to this endpoint but never reflected in the tests.